ci-curriculum.md

```diff
@@ -0,0 +1,82 @@
+# CI Curriculum - Community Outreach Program
```

Comment:
Let's have a very simple curriculum for the students , let's keep it structure.

Reply:
Need review from ci-engineering.